SimplenoteWidgets/Tools/DemoContent.swift

```diff
@@ -4,6 +4,22 @@ struct DemoContent {
     static let singleNoteTitle = "Twelve Tone Serialism"
     static let singleNoteContent = "The twelve-tone technique is often closely related with the compositional style, serialism. Fundamentally, twelve-tone serialism is a compositional technique where all 12 notes of the chromatic scale are played with equal frequency throughout the piece without any emphasis on any one note. For this reason, twelve-tone serialism avoids being in any key. Arnold Schoenberg was arguably the most influential composers who embraced the twelve-tone technique. Schoenberg described the system as a “Method of composing with twelve tones which are related only with one another.”"
     static let singleNoteContentAlt = "- [] Check item one \n- [] check item two\n- [] check item three"
+    static let singleNoteURL = URL(string: SimplenoteConstants.simplenoteScheme + "://")!
 
     static let demoURL = URL(string: SimplenoteConstants.simplenoteScheme + "://")!
+
+    static let listTag = "Composition"
```

Comment:
Should this string be localized?

Reply:
Differing this the [conversation on the Note Widget PR](https://github.com/Automattic/simplenote-ios/pull/1375#discussion_r693405169) about what these strings should be.